client/src/reset.scss

```diff
@@ -0,0 +1,329 @@
+/* Improve media defaults */
```

Comment:
Are you sure you want to apply this reset to all the pages?

Reply:
Yes, it covers potential compatibility issues across different browsers.